### PR TITLE
Replacing PyMC3 plots w/ Arviz plots & sigma Param change [Part 9]

### PR DESCRIPTION
**Pull request by CloudChaoszero**
# Description
The following is a large PR breakdown of PR #16.

Replace PyMC3 dependent plots with arviz plots in case studies & examples.
* The issue is a 3-part step for the following:
  * Initiative via Issue https://github.com/pymc-devs/pymc3/issues/4371 (Leave open)

Replace parameter `sd` with `sigma` (e.g. some examples have `pm.Normal(...sd=...)`
* Closes issue #2

### CODE REVIEW

File: examples/variational_inference/empirical-approx-overview.ipynb [hunk 1]
```diff
@@ -18,11 +18,13 @@
      "name": "stdout",
```
Comment (twiecki) [inline examples/variational_inference/empirical-approx-overview.ipynb:18]: <p>Shouldn't ignore warnings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (twiecki) [inline examples/variational_inference/empirical-approx-overview.ipynb:18]: <p>This looks odd. I think more recent arviz would plot them better.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (twiecki) [inline examples/variational_inference/empirical-approx-overview.ipynb:18]: <p>Missing output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (OriolAbril) [reply examples/variational_inference/empirical-approx-overview.ipynb:18]: <p>This is probably right, we improved the kde a couple releases back, the original implementation was too smoothing and did quite a bad job for multimodal and mixture distributions. The PR with the changes has some comparisons: https://github.com/arviz-devs/arviz/pull/1284</p>

Comment (CloudChaoszero) [reply examples/variational_inference/empirical-approx-overview.ipynb]: <p>Sounds great, thanks for the reference. Updated arviz dependency now combines the two into 1 graph.</p>

Comment (CloudChaoszero) [reply examples/variational_inference/empirical-approx-overview.ipynb]: <p>Done! </p><p> </p><p>Note:</p><p>There was an update for the <code>sns.kdeplot </code>plot functionality, and created an error.</p><p> </p><p>I went ahead and provided the same visualization by creating the 100x2 array into a DataFrame, and specified index in <code>kdeplot</code> params.</p>

**Pull request merged**